app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            <LinkButton title='Click to copy' className='version-text' onClick={this.onClickVersion}>Version {version}</LinkButton> ({releaseNotesLink})
```

Comment:
Ideally, the text should just be selectable. I am not a fan of having links that don't behave like links and having actions that don't provide the user with much feedback.